mods/d2k/rules/structures.yaml

```diff
@@ -194,10 +194,11 @@ refinery:
 		Capacity: 2000
 	CustomSellValue:
 		Value: 500
-	FreeActor:
+	FreeActorWithDelivery:
 		Actor: harvester
 		InitialActivity: FindResources
-		SpawnOffset: 2,1
+		DeliveryOffset: 2,1
```

Comment:
Can you tweak this a bit so that the carryall does not glitch into the refinery while descending?